### PR TITLE
Log socket path on running 'tns debug ios'

### PR DESCRIPTION
**Pull request by ivanbuhov**
This change is necessary for NativeScript VSCode plugin.

### CODE REVIEW

**Comment by ivanbuhov** [comment]
ping @rosen-vladimirov

File: lib/device-sockets/ios/socket-proxy-factory.ts [hunk 1]
```diff
@@ -119,6 +119,7 @@ export class SocketProxyFactory implements ISocketProxyFactory {
 
 		let socketFileLocation = temp.path({ suffix: ".sock" });
 		server.listen(socketFileLocation);
+		this.$logger.info("socket-file-location: " + socketFileLocation);
```
Comment (rosen-vladimirov) [inline lib/device-sockets/ios/socket-proxy-factory.ts]: as far as I understand, we need this only when `this.$options.client` is false (`--no-client` is passed)
When started from CLI, the --no-client will not be passed and users should not see this row?

If my understanding is correct, can you add the if, something like:
```TypeScript
if(!this.$options.client)  {
    this.$logger.info("socket-file-location: " + socketFileLocation);
}
```
and in the constructor:
```TypeScript
constructor(private $logger: ILogger,
 		private $projectData: IProjectData,
 		private $projectDataService: IProjectDataService,
 		private $options: IOptions) { }
```

**Comment by ivanbuhov** [comment]
You are right, the extension always runs `tns debug` with `--no-client`. I will fix it ;)

File: lib/device-sockets/ios/socket-proxy-factory.ts (hunk 1 above)
Comment (ivanbuhov) [inline lib/device-sockets/ios/socket-proxy-factory.ts]: You are right, the extension always runs tns debug with --no-client. I will fix it ;)

**Comment by rosen-vladimirov** [comment]
:+1:  after green build

**Pull request merged**